lib/sanitize.ts

```
@@ -0,0 +1,67 @@
+    ALLOWED_ATTR: [] // No attributes allowed
+  });
+}
+
+/**
+ * Sanitizes a JavaScript object by applying sanitizeString to all string properties.
+ * This is useful for sanitizing objects before they are serialized to JSON.
+ * 
+ * @param obj - The object to sanitize
+ * @returns A new object with all string properties sanitized
+ */
+export function sanitizeObject<T extends Record<string, any>>(obj: T): T {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)